docs/installation.rst

```diff
@@ -482,72 +482,67 @@ connect to the databases you want to access through Superset.
-+------------------+---------------------------------------+-------------------------------------------------+
-| Teradata         | ``pip install sqlalchemy-teradata``   | ``teradata://``                                 |
-+------------------+---------------------------------------+-------------------------------------------------+
-| Vertica          | ``pip install                         |  ``vertica+vertica_python://``                  |
-|                  | sqlalchemy-vertica-python``           |                                                 |
-+------------------+---------------------------------------+-------------------------------------------------+
-| Hana             | ``pip install hdbcli sqlalchemy-hana``|  ``hana://``                                    |
-|                  | or                                    |                                                 |
-|                  | ``pip install apache-superset[hana]`` |                                                 |
-+------------------+---------------------------------------+-------------------------------------------------+
++------------------+-------------------------------------------------------------------+-------------------------------------------------+
+| database         | pypi package                                                      | SQLAlchemy URI prefix                           |
++==================+===================================================================+=================================================+
+| Amazon Athena    | ``pip install "apache-superset[athena]"``                         | ``awsathena+jdbc://``                           |
++------------------+-------------------------------------------------------------------+-------------------------------------------------+
+| Amazon Redshift  | ``pip install  "apache-superset[redshift]"``                      | ``redshift+psycopg2://``                        |
++------------------+-------------------------------------------------------------------+-------------------------------------------------+
+| Apache Drill     | ``pip install "apache-superset[drill]"``                          | For the REST API:``                             |
+|                  |                                                                   | ``drill+sadrill://``                            |
+|                  |                                                                   | For JDBC                                        |
+|                  |                                                                   | ``drill+jdbc://``                               |
++------------------+-------------------------------------------------------------------+-------------------------------------------------+
+| Apache Druid     | ``pip install "apache-superset[druid]"``                          | ``druid://``                                    |
```

Comment:
@villebro what are your thoughts about removing the `pip install` prefix here and just mentioning this as a primer, i.e., to install the PyPI package run `pip install <package>`.

Reply:
I agree @john-bodley , currently it is overly repetitive.